deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,48 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+* {
+    padding: 0;
+    margin: 0;
+}
+
+
+.main-container-assignment-2 {
+    display: flex;
+    flex-direction: row;
+    height: 100vh;
+    flex-basis: 100vh;
+}
+
+.col-2 {
+    display: flex;
+    flex-direction: column;
+    width: calc(100vh - 200px);
+    height: 100vh;
+}
+
+.header-item {
+    flex-basis: 200px;
+    flex-grow: 0;
+    flex-shrink: 0;
+    width: calc(100vw - 200px);
```

Comment:
Width properties hoef je niet te gebruiken op items in de flexbox. Door een flex-basis in te stellen zou dat voldoende moeten zijn (in uitwerkingen is flex-basis: 20vw; gebruikt maar volgens mij is 200px ook genoeg)

Reply:
Dit geld voor alle width properties die je hebt gebruikt in deze opdrachten